### PR TITLE
Document our meetings process

### PR DESCRIPTION
**Pull request by nicks**
Hello @victorwuky, @dbentley,

Please review the following commits I made in branch nicks/holacracy:

5b184cb0f2ab4b8ae8ac35b534a7bc2bad222c6f (2020-05-29 11:20:26 -0400)
Document our meetings process
Document the goals of our meeting and written comms processes

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by nicks** [comment]
Most of the section on meeting structure is copied and pasted from the Working Agreement and has been there since at least 2018

Most of the section on goals is taken from the Working Agreement, and expanded a bunch to help clarify what it means in practice

File: communications/README.md [hunk 1]
```diff
@@ -1,9 +1,163 @@
...
+
+- We shouldn't weigh all perspectives equally in all situations. e.g., if we're
+  estimating a design task, we wouldn't ask the whole team to estimate it and
+  average the results.
+
+### To Understand Decisions
+
+#### What This Means
+
+- When Tilters leave a meeting, they should understand what the next steps are.
```
Comment (victorwuky) [inline communications/README.md:48]:
```suggestion
- When Tilters read something (e.g. Clubhouse comment) or leave a meeting, they should understand what the next steps are.
```

File: communications/README.md [hunk 2]
```diff
@@ -1,9 +1,163 @@
...
+Ensure your [Clubhouse notifications](https://help.clubhouse.io/hc/en-us/sections/201543416-Notifications) are properly configured.
+
+### GitHub
+
+We use GitHub as the main system of record for:
+- Code review and low-level discussion of changes
+- Processes we use at Tilt (in repos like [this one](https://github.com/tilt-dev/company))
+- Public-facing bugs and feature requests
+
+If you want to discuss a proces s change, consider the smallest reasonable
```
Comment (victorwuky) [inline communications/README.md:152]:
```suggestion
If you want to discuss a process change, consider the smallest reasonable
```

Comment (nicks) [reply communications/README.md:152]: done

**Comment by nicks** [comment]
@landism @maiamcc @ellenkorbes @hyu @jazzdan happy to accept any other comments or revisions! Otherwise I'll merge this end of day Monday

File: communications/README.md [hunk 3]
```diff
@@ -1,9 +1,163 @@
 # Communications
 
-## Async written
+## Goals
```
[the remark this answers is not in the document]
Comment (victorwuky) [reply communications/README.md:3]: Okay cool.

**Pull request merged**